pkg/arvo/sys/vane/eyre.hoon

```diff
@@ -2518,13 +2518,13 @@
       :_  state
       :_  ~
       ^-  move
-      =/  as=^ship
-        ?+  t.t.extra  ~|([%strange-wire extra] !!)
-          ~      our  ::  old-style wire
-          [@ ~]  (slav %p i.t.t.extra)
+      =+  ?+  t.t.extra  ~|([%strange-wire extra] !!)
+          ~      [as=our is-old-wire=%.y]
+          [@ ~]  [as=(slav %p i.t.t.extra) is-old-wire=%.n]
         ==
+      =/  =wire  (subscription-wire channel-id request-id as ship app)
       %+  deal-as
-        (subscription-wire channel-id request-id as ship app)
+        ?:  is-old-wire  (snip wire)  wire
```

Comment:
Worth leaving a comment.

```suggestion
        ::NOTE  we previously used a wire format that had the local identity
        ::      implicit, instead of explicit at the end of the wire. if we
        ::      detect we used the old wire here, we must re-use that format
        ::      (without id in the wire) for sending the %leave.
        ?:(old (snip wire) wire)
```